src/Mod/Sketcher/TestSketchFillet.py

```diff
@@ -0,0 +1,291 @@
+#   (c) Juergen Riegel (FreeCAD@juergen-riegel.net) 2011      LGPL        *
```

Comment:
Presumably this is not really true: replace with the actual copyright info (e.g. yours!)